sections/semantics-embedded-content.include

```diff
@@ -7539,8 +7509,9 @@ zero or more <{track}> elements, then
   The <dfn attribute for="HTMLMediaElement"><code>currentTime</code></dfn> attribute must, on
   getting, return the <a href="#media-elements">media element</a>'s <a>default playback start position</a>,
   unless that is zero, in which case it must return the element's <a>official playback
-  position</a>. The returned value must be expressed in seconds. On setting, if the <a href="#media-elements">media element</a> has a <a>current media controller</a>, then the user agent must throw an
-  <code>InvalidStateError</code> exception; otherwise, if the <a href="#media-elements">media element</a>'s <code>readyState</code> is <code>HAVE_NOTHING</code>, then it must set the <a href="#media-elements">media element</a>'s <a>default playback start position</a> to the new value; otherwise, it must
+  position</a>. The returned value must be expressed in seconds. If the
```

Comment:
You need to keep the "on setting" part, which appears to have been accidentally dropped.